controllers/part/part_ctlr.go

```diff
@@ -77,8 +79,45 @@ func All(w http.ResponseWriter, r *http.Request, params martini.Params, enc enco
 			count = ct
 		}
 	}
+	//Convert given dates to ISO8601 format.
+	if qs.Get("modified-from") != "" {
+		from = qs.Get("modified-from")
+		_, err := time.Parse(time.RFC3339, from)
+		if err != nil {
+			_, err := time.Parse(time.RFC3339, from+"-06:00")
```

Comment:
It's mostly like this because the time.Parse function is a bit rigid, if the input doesn't match up exactly to the specified format, then it fails. Since this is probably just being used by Perficient I suppose it would be fine to make it so that they have to specify the full timestamp each time if we wanted to do so, just thought I'd add a few default values in there so that the date format could be a bit flexible.